test/unit/lib/TopStoriesFeed.test.js

```diff
@@ -1460,4 +1460,38 @@ describe("Top Stories Feed", () => {
     assert.calledOnce(instance.uninit);
     assert.calledOnce(instance.init);
   });
+  describe("#layout", () => {
+    it("should call dispatchLayoutUpdate from loadCachedData and fetchStories", async () => {
```

Comment:
For a single test, this is pretty complex.  I'd suggest splitting it in two.

Reply:
done

Reply:
Thanks.